bin/run-browser-tests-ci-batch

```diff
@@ -0,0 +1,42 @@
+#! /usr/bin/env python3
+
+# DOC: Run a batch of browser tests in CI mode. Takes one argument, the
+# DOC: 1-indexed batch number.
+
+import glob
+import math
+import subprocess
+import sys
+
+BATCH_COUNT = 4
+
+def bad_batch_number_input(batch_num_input):
+    print(f"Invalid batch number: {batch_num_input}", file=sys.stderr)
+    exit(1)
+
+if len(sys.argv) < 2:
+    bad_batch_number_input("none")
+
+try:
+    batch_number = int(sys.argv[1])
+except ValueError:
+    bad_batch_number_input(sys.argv[1])
+
+if batch_number < 1 or batch_number > BATCH_COUNT:
+    bad_batch_number_input(batch_number)
+
+test_files = glob.glob('browser-test/src/*.test.ts')
+test_files = [file.replace("browser-test/", "") for file in test_files]
+test_files.sort()
+test_files_count = len(glob.glob('browser-test/src/*.test.ts'))
```

Comment:
Just curious, any reason to do this instead of just length of test_files?

Reply:
nope, that was a mistake. thanks!